docs/update_openshift.md

```diff
@@ -80,15 +97,51 @@ Upgrading database and Operator on [Red Hat Marketplace :octicons-link-external-
 
         Click the "Upgrade available" link to see upgrade details, then click "Preview InstallPlan" button, and finally "Approve" to upgrade the Operator.
 
-=== "Operator 1.15.0 and newer"
+### Update via the command-line interface
 
-    You can actually update the Operator via the [Operator Lifecycle Manager (OLM) :octicons-link-external-16:](https://docs.redhat.com/en/documentation/openshift_container_platform/4.2/html/operators/understanding-the-operator-lifecycle-manager-olm#olm-overview_olm-understanding-olm) web interface.
+1. Check all clusters managed by the Operator to see if `initContainer.image` is set.
 
-    Login to your OLM installation and list installed Operators for your Namespace to see if there are upgradable items:
+        * If defined: skip the next step.
```

Comment:
The below steps are needed only for those, who are planning to use Openshift 4.19. Maybe we should mention that?